### DISCUSSION TITLE
Create a Wish/Watch list

### DISCUSSION THREAD

**Discussion by Nelinski** [sct/overseerr#2790] [Feature Requests]
I'd like to be able to flag movies/tv shows on a "watch list" to come back to at a later date once it's released and there are reviews. I don't want to request it and it just take up space on my server if it has bad reviews, but I also don't want to forget about it. Having a "add to watch list" button would be great.

**Comment by garbled1** [comment]
I would really like this feature.  I like to buy DVD's/Blu-rays of things, and rip them myself.  Ideally I would like to have a way to just track those with a list of "oh hey, buy these"

**Comment by Drallas** [comment]
I also like to have this too! Sometimes you just want to park it for later.
